### PR TITLE
update cadproc to accept iso 20022 json format

### PR DESCRIPTION
**Pull request by uche-lextego**

### CODE REVIEW

**Comment by theolextego** [comment]
The RuleRequest class, NetworkMap class, TypologyResult class all need to be updated.

**Review by theolextego — CHANGES_REQUESTED** [review]
The RuleRequest class, NetworkMap class, TypologyResult class all need to be updated.

**Review by theolextego — APPROVED** [review]

File: .env.template [hunk 1]
```diff
@@ -1,14 +0,0 @@
-REST_PORT=3000
```
Comment (johanfol) [inline .env.template:1]: Why did you delete the .env.template file? This must never be deleted - if anyone else now needs to work on this code, it will be very difficult what to configure.  When devops needs to give this service all the config in jenkins, it will be almost impossible for them. Please add this file back - but ensure in this file that there are no sensitive information.

**Review by johanfol — CHANGES_REQUESTED** [review]

File: .env.template (hunk 1 above)
Comment (uche-lextego) [reply .env.template:1]: Ouch. my bad! must have forgotten to revert. Thanks
